### PR TITLE
Refactor namings

### PR DESCRIPTION
**Pull request by clemera**

### CODE REVIEW

**Comment by MatthewZMD** [comment]
Coincidentally, there is another patch from Stefen Monnier that will be resolving some of the prefix renames, I am about to create a PR for him (he doesn't use Github) but I don't have enough time atm. 

Since you have already done this, can we set the naming convention for prefixes to `eaf--` instead of `eaf-` for internally-used stuff?

File: eaf.el [hunk 1]
```diff
@@ -94,6 +94,9 @@
     map)
   "Keymap used by `eaf-mode'.")
 
+(defvar-local eaf-buffer-id nil
```
Comment (MatthewZMD) [inline eaf.el]: As per comment, change to `eaf--buffer-id` instead

**Comment by MatthewZMD** [comment]
Also, on the topic of this, can you rename `buffer-url` into `eaf--buffer-url` and `buffer-app-name` into `eaf--buffer-app-name` too?

**Comment by clemera** [comment]
> can we set the naming convention for prefixes to eaf-- instead of eaf- for internally-used stuff?

+1, I followed the present naming conventions, I will do the other renames, too.

**Comment by manateelazycat** [comment]
Thanks for patch.

**Pull request merged**